core/state/pruner/pruner.go

```diff
@@ -62,6 +63,8 @@ type Config struct {
 	BloomSize      uint64 // The Megabytes of memory allocated to bloom-filter
 	Threads        int    // The maximum number of threads spawned in dumpRawTrieDescendants and removeOtherRoots
 	CleanCacheSize int    // The Megabytes of clean cache size used in dumpRawTrieDescendants
+
+	ParallelPrune bool // Whether to prune in parallel per account
```

Comment:
Can we name it a bit different to specify that it only disables/enables parallelization of account storage traversal? something like `ParralelStorageTraversal` (or something shorter 😄 )

Otherwise, if user sets `ParallelPrune = false` they might expect that there won't be any parallelization of pruning, but traversing accounts trie still will be run in multiple goroutines (if `Threads` > 0, and by default it is `runtime.NumCPU()` if I am not mistaken)

Reply:
Done.